cores/arduino/time.cpp

```diff
@@ -17,20 +19,28 @@ __attribute__((weak)) void yield() {
 }
 
 static FspTimer main_timer;
+const uint8_t _timer_type = AGT_TIMER;
```

Comment:
Hi @Kees-van-der-Oord ☕ 👋 

Thank you for your good work, but there has been a misunderstanding. I did not ask you to replace the macros with constants or inline functions. Instead I meant that you should directly inline them into the calling code, that is to say here:

```C++
main_timer.begin(
  TIMER_MODE_PERIODIC,
  /* _timer_type */ AGT_TIMER,
  /* _timer_index */ 0,
  _timer_period,
  1,
  /* _timer_clock_divider */ TIMER_SOURCE_DIV_8,
  timer_micros_callback);
```